internal/datastore/mysql/options.go

```diff
@@ -103,12 +104,13 @@ func TablePrefix(prefix string) Option {
 	}
 }
 
-// EnablePrometheusStats enables Prometheus metrics provided by Go's database/sql package
+// WithEnablePrometheusStats marks whether Prometheus metrics provided by Go's database/sql package
+// are enabled.
 //
-// Prometheus metrics are enabled by default.
-func EnablePrometheusStats() Option {
+// Prometheus metrics are disabled by default.
+func WithEnablePrometheusStats(enablePrometheusStats bool) Option {
```

Comment:
naming nit: perhaps `WithPrometheusStats` would be easier to reason about?

Reply:
I vacillated back and forth on that, but I decided to be explicit